Game/Src/TestEnemyMovement.gd

```diff
@@ -241,14 +247,14 @@ func lightning():
 		closest_enemy.die()
 
 func slippery_hands():
-	print(GlobalVariables.glory)
-	GlobalVariables.glory -= 0.05
-	print(GlobalVariables.glory)
+	emit_signal("glory_increased",-10)
 	pass
 
 func potion():
 	emit_signal('potion')
-	pass
+	var playerHealth = characterStats.get_health()
+	print('current health ',playerHealth)
+	characterStats.set_health(playerHealth+10)
```

Comment:
To fine tune the balance of the game, it would be nice to have all the effects of powers up/down in a single file.
So we'll have potion_hp_gain and slippery_gory_lost then import these variables here and use then in the emits
Not sure how to do that but we should make a note to do it at some points.

Reply:
Indeed, I worked a quick and dirty way first to test some functions